### PR TITLE
Feature/ui fixes

### PR DESCRIPTION
**Pull request by leborgneromain**

### CODE REVIEW

File: client-ui/src/app/components/footer/footer.component.ts [hunk 1]
```diff
@@ -13,15 +13,21 @@
...
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.less']
+})
+export class FooterComponent {
+
+  constructor(
+  ) { }
+
+  appVersion: string = __APP_VERSION__
```
Comment (wajda) [inline client-ui/src/app/components/footer/footer.component.ts]: Codacy Issue found: [The class property 'appVersion' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4483640)

File: client-ui/extra-webpack.config.js [hunk 2]
```diff
@@ -14,9 +14,12 @@
  * limitations under the License.
  */
 
-cytoscape-ng {
-    height          : 96vh;
-    float           : left;
-    width           : 100%;
-    background-color: #FFF;
+const webpack = require('webpack');
```
Comment (wajda) [inline client-ui/extra-webpack.config.js]: Codacy Issue found: [Extra semicolon. (semi)](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4483640)

File: client-ui/src/index.html [hunk 3]
```diff
@@ -30,7 +30,8 @@
   // Uncomment it to set Spline client configuration properties:
 
   window['SplineConfiguration'] = {
-    apiUrl: "PUT_YOUR_SPLINE_REST_ENDPOINT_URL_HERE"
+    apiUrl: "PUT_YOUR_SPLINE_REST_ENDPOINT_URL_HERE",
+    embeddedMode : true => hide header / false => display full app with header
```
Comment (wajda) [inline client-ui/src/index.html]: Won't it break our Client Web app?
See `WebController.index(...)`  method. It replaces the placeholder and uncomments this block.

Comment (leborgneromain) [reply client-ui/src/index.html]: Well spotted. Thanks

File: client-ui/extra-webpack.config.js (hunk 2 above)
Comment (wajda) [inline client-ui/extra-webpack.config.js]: Codacy Issue found: [Extra semicolon. (semi)](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4483640)

File: client-ui/src/app/components/footer/footer.component.ts (hunk 1 above)
Comment (wajda) [inline client-ui/src/app/components/footer/footer.component.ts]: Codacy Issue found: [The class property 'appVersion' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/ABSA_OSS/spline/pullRequest?prid=4483640)

**Pull request merged**